save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/vulnerability/VulnerabilityDto.kt

```diff
@@ -0,0 +1,17 @@
+package com.saveourtool.save.entities.vulnerability
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property name of vulnerability
+ * @property progress critical of vulnerability
```

Comment:
Can you please describe `progress` in more detail?

What is it?
What values/ranges are acceptable?
If, let's say, only values between `0..100` are acceptable, please add the corresponding `require()` calls to `init {}`.

Reply:
This data is taken from an external database. I don't think we should throw an error if we get bad data.

Reply:
Why shouldn't we? To my mind, it is better to display only valid data anyway